src/test/java/com/epam/esm/persistence/jdbc/JdbcTemplateTagDaoImplTest.java

```diff
@@ -0,0 +1,119 @@
+package com.epam.esm.persistence.jdbc;
+
+import com.epam.esm.TestConfig;
+import com.epam.esm.persistence.dao.Tag;
+import com.epam.esm.persistence.jdbc.tag.JdbcTemplateTagDao;
+import com.epam.esm.persistence.util.mapper.TagMapperDB;
+import com.epam.esm.util.validation.BaseTagValidator;
+import com.epam.esm.web.exception.EntityBadInputException;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.jdbc.Sql;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.springframework.test.context.support.AnnotationConfigContextLoader;
+
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@ExtendWith(SpringExtension.class)
+@ContextConfiguration(classes = { TestConfig.class }, loader = AnnotationConfigContextLoader.class)
+@Sql(scripts = "classpath:sql/db-init.sql", executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD)
+@Sql(scripts = {"classpath:sql/db-clean.sql"}, executionPhase = Sql.ExecutionPhase.AFTER_TEST_METHOD)
+class JdbcTemplateTagDaoImplTest {
+
+    private final JdbcTemplateTagDao jdbcTemplateTagDao;
+
+    @Autowired
+    public JdbcTemplateTagDaoImplTest(JdbcTemplate jdbcTemplate,
+                                      BaseTagValidator<Tag, Long> tagValidation,
+                                      TagMapperDB tagMapperDB) {
+        this.jdbcTemplateTagDao = new JdbcTemplateTagDao(jdbcTemplate, tagValidation, tagMapperDB);
+    }
+
+    @Test
+    void findAllTags() {
+        int sizeAfterInit = 6;
+
+        List<Tag> listResult = jdbcTemplateTagDao.findAllEntities();
+
+        assertEquals(sizeAfterInit, listResult.size());
+    }
+
+    @Test
+    void findTagByIdExists() {
```

Comment:
I'm not quite sure how this test is working fine.... We have a `deleteSuccess` test method below, which deletes a certificate with `1L` id. So do these test methods depend on test run order?